### ISSUE TITLE
ExecutionTime - timing is not right, outputs to one cell whenn all cells are running

### ISSUE THREAD

**Issue by denfromufa** [ipython-contrib/IPython-notebook-extensions#469]
This is what happens on second cell:

```
Lastly executed on Wed, Feb 10 2016 at 10:58 AM in 0 s in 1 s in 0 s in 1 s in 0 s in 0 s in 0 s in 0 s in 0 s in 1 s in 0 s in 39 s in 0 s in 0 s in 0 s in 0 s in 2 s in 0 s in 0 s in 0 s in 2 s in 2 s in 1 s in 1 s in 1 s in 0 s in 0 s in 0 s in 0 s in 0 s in 0 s in 2 s in 0 s in 0 s in 0 s in 1 s in 0 s in 0 s in 1 s in 0 s in 15 s in 2 s in 0 s in 0 s in 0 s in 0 s in 1 s in 1 s in 1 s in 2 s in 0 s in 1 s in 0 s in 0 s in 1 s in 1 s in 1 s in 1 s in 0 s in 0 s in 0 s in 1 s in 0 s in 1 s in 0 s in 0 s in 0 s in 1 s in 0 s in 0 s in 1 s in 0 s in 1 s in 1 s in 1 s in 1 s in 22 min 23 s
```

**Comment by denfromufa** [comment]
@juhasch

**Comment by denfromufa** [comment]
The version of the notebook server is 4.1.0 and is running on:
Python 2.7.9 |Continuum Analytics, Inc.| (default, Dec 12 2014, 14:56:54) [MSC v.1500 32 bit (Intel)]

**Comment by jcb91** [comment]
This sounds like a duplicate of #436. I've recently done a rewrite of execute_timings (see #470) which would fix this, if you'd like to try that but if you'd like me to try to fix the existing implementation, @denfromufa, you could take a look at my reply to #436. I take your comments above to mean that:
1. You execute a bunch of cells (2? more than 2?) in one call (or queue several before the first execution is complete)
2. Whenever any cell completes, `'in 1 s'` (or similar) is added to the timing text on the _second_ (is this the last?) cell _instead of_ (as well as?) the appropriate cell.

**Comment by lucasb-eyer** [comment]
While I'm not @denfromufa, I came here to report the same issue. I tried #470 and it seems to fix all of these issues, thanks a lot!

Just for clarification:

1. Any amount more than one cell. Can be queued by "Run all"-style commands, or Ctrl/Shift-enter while another one still runs. All ways trigger the problem.
2. Added to the wrong cell *instead of* the appropriate one. Which one is complicated, it seems there's a type of queue as to which, but that queue gets (and stays) off-by-N where N is the amount (give or take 1) which had been queued originally.

But again, your PR #470 fixes all this, great work!
